jormungandr/src/blockchain/chain.rs

```diff
@@ -48,7 +48,7 @@ See Internal documentation for more details: doc/internal_design.md
 [`Ref`]: ./struct.Ref.html
 [`Branch`]: ./struct.Branch.html
 */
-
+#![allow(clippy::large_enum_variant)]
```

Comment:
I would rather override this on every enum type specifically. We should think of boxing some oversized fields.

Reply:
It doesn't work in the enum type as I posted last week. But I can box the fields.

Reply:
OK to leave as is then. Boxing needs to be done carefully, that's why I don't object to overrides themselves.